src/store/reducers/executeTopQueries/executeTopQueries.ts

```diff
@@ -49,6 +48,25 @@ LIMIT 100
 `;
 };
 
+function getRunningQueriesText(path: string, filters?: TopQueriesFilters, sortOrder?: SortOrder[]) {
+    const filterConditions = filters?.text
+        ? `Query ILIKE '%${filters.text}%' OR UserSID ILIKE '%${filters.text}%'`
+        : '';
+
+    const orderBy = prepareOrderByFromTableSort(sortOrder);
+
+    return `${QUERY_TECHNICAL_MARK}
+SELECT
+    UserSID, 
+    QueryStartAt, 
+    Query as QueryText, 
+    ApplicationName
+FROM \`${path}/.sys/query_sessions\`
```

Comment:
In case of nested DB previous query always returned running queries of domain database, so I added path